package.json

```diff
@@ -42,6 +42,7 @@
         "storybook": "7.2.0"
     },
     "devDependencies": {
+        "@hopper-ui/tokens": "0.1.0-beta.0",
```

Comment:
Having this dependency, like that will make it depends on previous version, should you get the current one instead using [Workspace Packages](https://pnpm.io/workspaces#publishing-workspace-packages) PNPM feature?